eth/ethereum.go

```diff
@@ -349,3 +350,9 @@ func (c *EthereumClient) EthCall(ctx context.Context, tx *types.Transaction,
 	result, err := c.client.CallContract(ctx, msg, blockNum)
 	return result, tracerr.Wrap(err)
 }
+
+// EthTransaction returns the transaction information of the given txHash
```

Comment:
Is this function really needed? It seems, that you don't use it anywhere

Reply:
Thank you I forgot to remove the unused code